EMML/1-1000/EMML150.xml

```diff
@@ -46,8 +46,10 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                         </support>
                         <extent>
                            <measure unit="leaf">ff 76+5</measure>
-                           <dimensions>
-                              <dim>33.5 x 29 cm</dim>
+                           <dimensions type="outer" unit="mm">
+                             <height>335<height>
```

Comment:
sorry I did not notice here: you have to open tag as
`<height>` and close it as `</height>`
always a slash in the end tag!
same for `</width>` at the end